docs/design.md

```diff
@@ -178,6 +176,14 @@ A Runner `Pod` has the following state as a GitHub Actions job runner.
 - `running`: `Pod` is running. Registered in GitHub Actions.
 - `debugging`: The job has finished with failure and Users can enter `Pod` to debug.
 
+In addition, it has the following states as the exit state of the execution result of `Runner.Listener`.
+
+- `successful`: When the job is successful and the RunnerPod will continue to finish.
+- `terminated_error`: When the job is a failure and the RunnerPod will continue to finish.
```

Comment:
terminated_error seems to be returned when a fatal error occurs.

https://github.com/actions/runner/blob/be9632302ceef50bfb36ea998cea9c94c75e5d4d/src/Runner.Listener/Program.cs#L47